egs/wsj/s5/local/nnet3/run_lstm.sh

```diff
@@ -24,6 +24,7 @@ non_recurrent_projection_dim=256
 chunk_width=20
 chunk_left_context=40
 chunk_right_context=0
+ivector_interval=10
```

Comment:
also pass this value to run_ivector_common.sh script, where it will be used as an argument to steps/online/nnet2/extract_ivectors_online.sh script.